webapps-schema/src/main/resources/schema/elasticsearch/create/index/tasklist-metric.json

```diff
@@ -0,0 +1,23 @@
+{
+  "mappings": {
+    "dynamic": "strict",
+    "properties": {
+      "id": {
+        "type": "keyword"
+      },
+      "eventTime": {
+        "format": "date_time || epoch_millis",
+        "type": "date"
+      },
+      "event": {
+        "type": "keyword"
+      },
+      "value": {
+        "type": "keyword"
+      },
+      "tenantId": {
```

Comment:
❓ Why do we have this?

Reply:
Not sure how useful the tenant is for the metrics 🤔

Reply:
It's the same as in Operate's metric index. Not really why we would need the tenant there as well.